### PR TITLE
Updating Onboarding Docs with VS Code Cluster Setup

### PR DESCRIPTION
**Pull request by SylLutze**
Would appreciate feedback on if I should frame this more strongly - "We recommend using VS code" instead of "here's another option!".

### CODE REVIEW

File: docs/source/model_design/programming_vivarium/debugging/index.rst [hunk 1]
```diff
@@ -202,9 +202,9 @@ these types of errors earlier and be able to fix them. To do this, we
 recommend having a file, or Jupyter notebook open to run some basic 
 checks.
 
-While working in VS Code with the cluster set-up - information on 
-doing this can be found in the Onboarding Resources section of 
-Vivarium Research - you can run Jupyter notebooks while creating 
+While working in VS Code with the cluster set-up, `information on this can be found here <https://stash.ihme.washington.edu/projects/CCGUTS/repos/script_utils/browse/bin/submit_vscode_server.py>`_ on there 
+is more in depth information in the Onboarding Resources section of 
+Vivarium Research, you can run Jupyter notebooks while creating
```
Comment (zmbc) [inline docs/source/model_design/programming_vivarium/debugging/index.rst:207]: I think the grammar of this sentence got a bit weird

**Review by zmbc — APPROVED** [review]
Looks good!

File: docs/source/model_design/programming_vivarium/debugging/index.rst [hunk 2]
```diff
@@ -202,9 +202,9 @@ these types of errors earlier and be able to fix them. To do this, we
 recommend having a file, or Jupyter notebook open to run some basic 
 checks.
 
-While working in VS Code with the cluster set-up - information on 
-doing this can be found in the Onboarding Resources section of 
-Vivarium Research - you can run Jupyter notebooks while creating 
+While working in VS Code with the cluster set-up, `information on this can be found here <https://stash.ihme.washington.edu/projects/CCGUTS/repos/script_utils/browse/bin/submit_vscode_server.py>`_ on there
```
Comment (aflaxman) [inline docs/source/model_design/programming_vivarium/debugging/index.rst:205]:
```suggestion
Information on working in VS Code with the cluster `is available here <https://stash.ihme.washington.edu/projects/CCGUTS/repos/script_utils/browse/bin/submit_vscode_server.py>`_ and there 
```

File: docs/source/model_design/programming_vivarium/debugging/index.rst [hunk 3]
```diff
@@ -202,9 +202,9 @@ these types of errors earlier and be able to fix them. To do this, we
 recommend having a file, or Jupyter notebook open to run some basic 
 checks.
 
-While working in VS Code with the cluster set-up - information on 
-doing this can be found in the Onboarding Resources section of 
-Vivarium Research - you can run Jupyter notebooks while creating 
+While working in VS Code with the cluster set-up, `information on this can be found here <https://stash.ihme.washington.edu/projects/CCGUTS/repos/script_utils/browse/bin/submit_vscode_server.py>`_ on there 
+is more in depth information in the Onboarding Resources section of
```
Comment (aflaxman) [inline docs/source/model_design/programming_vivarium/debugging/index.rst:206]:
```suggestion
is more in-depth information in the Onboarding Resources section of 
```

File: docs/source/model_design/programming_vivarium/debugging/index.rst (hunk 1 above)
Comment (aflaxman) [inline docs/source/model_design/programming_vivarium/debugging/index.rst:207]:
```suggestion
Vivarium Research [link].  You can run Jupyter notebooks while creating 
```

**Review by aflaxman — APPROVED** [review]

**Pull request merged**